meta/errors.go

```diff
@@ -26,6 +26,17 @@ var (
 	// ErrNodesRequired is returned when at least one node is required for an operation.
 	// This occurs when creating a shard group.
 	ErrNodesRequired = errors.New("at least one node required")
+
+	// ErrNodeIDRequired is returned when using a zero node id.
+	ErrNodeIDRequired = errors.New("node id must be greater than 0")
+
+	// ErrNodeUnableToDropSingleNode is returned if the node being dropped is the last
+	// node in the cluster
+	ErrNodeUnableToDropFinalNode = errors.New("unable to drop the final node in a cluster")
+
+	// ErrNodeDataLossImminent is returned if the node requested to be dropped has
+	// the last copy of a shard present and the force keyword was not used
+	ErrNodeDataLossImminent = errors.New("unable to drop node - data loss would occur - use the optional keyword 'FORCE' to force this removal")
```

Comment:
So I think this error needs to be more generic. E.g.

ErrShardNotReplicated = errors.New("shard not replicated")

Then it is up to the code receiving this error to return to the user information about the `FORCE` keyword. I'm not 100% about this, but adding hints about InfluxQL to error messages doesn't seem like the right thing to do.

`ErrNodeUnableToDropFinalNode` is much better, by comparison.